Dockerfile

```diff
@@ -21,7 +21,28 @@ RUN mkdir -p /opt/fits && \
   curl -fSL -o /opt/fits-1.0.5.zip http://projects.iq.harvard.edu/files/fits/files/fits-1.0.5.zip && \
   cd /opt && unzip fits-1.0.5.zip && chmod +X fits-1.0.5/fits.sh
 
-RUN mkdir /data
+RUN apt-get update && \
+      apt-get -y install sudo
+
+ARG UID=8083
+ARG GID=8083
+
+# Create the home directory for the new app user.
+RUN mkdir -p /data
+RUN chown ${UID}:${GID} /data
+
+# Create an app user so our program doesn't run as root.
+RUN groupadd -g ${GID} app
+RUN useradd -g ${GID} -l -M -s /bin/false -u ${UID} app
```

Comment:
Setting the default command to `/bin/false` seems weird to me; I've not seen this done for user accounts inside images before.  I think we should just stick with existing patterns:

- https://github.com/docker-solr/docker-solr/blob/master/7.7/Dockerfile#L32
- https://github.com/docker-library/mariadb/blob/master/10.3/Dockerfile#L4
- https://github.com/docker-library/redis/blob/master/5.0/Dockerfile#L3

There are also some images which start as root but change their user after doing some setup.  This may be an easier path for us, but I personally haven't looked into that approach.